stock_measuring_device/static/src/js/measuring_wizard.js

```diff
@@ -0,0 +1,62 @@
+odoo.define("stock_measuring_device.measuring_wizard", function(require) {
+    "use strict";
+
+    var FormController = require("web.FormController");
+
+    FormController.include({
+        init: function() {
+            this._super.apply(this, arguments);
+            if (this.modelName === "measuring.wizard") {
+                this.call(
+                    "bus_service",
+                    "addChannel",
+                    "notify_measuring_wizard_screen"
+                );
+                this.call(
+                    "bus_service",
+                    "on",
+                    "notification",
+                    this,
+                    this.measuring_wizard_bus_notification
+                );
+                this.call("bus_service", "startPolling");
+            }
+        },
+        measuring_wizard_bus_notification: function(notifications) {
+            var self = this;
+            _.each(notifications, function(notification) {
+                var channel = notification[0];
+                var message = notification[1];
+                if (channel === "notify_measuring_wizard_screen") {
+                    switch (message.action) {
```

Comment:
this switch seems useless as you are checking only one condition which can be done w/ an IF